home.html

```diff
@@ -22,24 +22,30 @@
       <nav>
         <!-- Waiting for logo here -->
         <div class="nav-links" id="navLinks">
-          <i class="fa fa-times" onclick="hideMenu()"></i>
           <ul>
             <li><a href="home.html">Accueil</a></li>
             <li><a href="about.html">A Propos de nous</a></li>
             <li><a href="services.html">Nos services</a></li>
             <li><a href="blog.html">Activités</a></li>
-            <li><a href="#">FaQ</a></li>
-            <li><a href="contact.html">Contacter Nous</a></li>
+            <li><a href="#">Foire A Questions</a></li>
           </ul>
         </div>
         <i class="fa fa-bars" onclick="showMenu()"></i>
+        <!-- <i class="fa-solid fa-xmark" onclick="closeMenu()"></i> -->
+        <a href="contact.html"
```

Comment:
@AmaniLouendriz it's rare to nest a `button` tag inside an `a` anchor tag. If you want it to look like a button, I recommend looking up how to style a link like a button. You can basically just have the link, and make it look like a button.

Reply:
Hello @caseyli , I deleted the "button" tag but I kept the same style for the class "my-button", is that correct? thank you so much !

Reply:
@AmaniLouendriz looks good!